src/utils/auth.js

```diff
@@ -0,0 +1,8 @@
+// middleware that checks required authentication
+function requireAuth(req, res, next) {
```

Comment:
Thank you for making this! This should be a good way to ensure requests are authorized